components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-[2px]";
+
+export const Footer = ({ links }: FooterProps) => {
+  return (
+    <footer
+      className="flex items-center justify-center bg-blue pb-14 pt-10 md:pt-[52px]"
```

Comment:
Let avoid exact (absolute) units at all costs, instead of `md:pt-[52px]` let's use `md:pt-12` (or `14`).

> If you are using VSCode I always suggest [Tailwind CSS IntelliSense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) extension.

Reply:
Also, here in the `footer` element, it appears to me that the classes `flex items-center justify-center` aren't doing anything